.github/dependabot.yml

```
@@ -4,5 +4,7 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-    time: "01:00"
+    time: "09:14"
```

```suggestion
    time: "08:40"
```
I would like to see whether this works and don't want to wait for another 34 min;)